diff -urN aaa/pygobject-2.11.2/gobject/pygobject.h pygobject-2.11.2/gobject/pygobject.h
--- aaa/pygobject-2.11.2/gobject/pygobject.h	2006-08-04 20:10:29.000000000 +0200
+++ pygobject-2.11.2/gobject/pygobject.h	2006-08-13 22:58:32.686268000 +0200
@@ -178,7 +178,7 @@
     void      (*add_warning_redirection) (const char *domain,
                                           PyObject   *warning);
     void      (*disable_warning_redirections) (void);
-    void      (*type_register_custom)(const gchar *typename,
+    void      (*type_register_custom)(const gchar *typename_,
 				      PyGTypeRegistrationFunction callback,
 				      gpointer data);
 
