--- pygobject-2.26.0/configure.ac~	2010-09-27 17:59:54.000000000 +0200
+++ pygobject-2.26.0/configure.ac	2010-10-19 13:59:45.988405664 +0200
@@ -174,6 +174,7 @@
 AM_PATH_GLIB_2_0(glib_required_version,,[AC_MSG_ERROR(maybe you want the pygobject-2-4 branch?)],gobject $extra_mods)
 if test -n "$export_dynamic"; then
   GLIB_LIBS=`echo $GLIB_LIBS | sed -e "s/$export_dynamic//"`
+  GLIB_LIBS="$GLIB_LIBS -lpython"
 fi
 PYTHON_BASENAME=`basename $PYTHON`
 AC_SUBST([PYTHON_BASENAME])
